<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Gray_histogram</class>
 <widget class="QWidget" name="Gray_histogram">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>直方图</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_all">
      <attribute name="title">
       <string>总体</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_r">
      <attribute name="title">
       <string>红色通道</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_g">
      <attribute name="title">
       <string>绿色通道</string>
      </attribute>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>570</x>
         <y>490</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_b">
      <attribute name="title">
       <string>蓝色通道</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
